benchmark_runner/common/oc/oc.py

```diff
@@ -244,6 +244,16 @@ def wait_for_dv_status(self,
             current_wait_time += OC.SLEEP_TIME
         raise DVStatusTimeout(status=status)
 
+    def verify_odf_installation(self):
+        """
+        This method verifies ODF installation
+        :return: True ODF passed, False failed
+        """
+        self.run(""" oc patch ocsinitialization ocsinit -n openshift-storage --type json --patch '[{ "op": "replace", "path": "/spec/enableCephTools", "value": true }]' """)
```

Comment:
Please use the storagecluster to enable the toolbox, We are planning to remove using the ocsinitialization cr for enabling the toolbox in 4.15 or some future release.
so you should replace your command to
oc patch storagecluster ocs-storagecluster -n openshift-storage --type json --patch '[{ "op": "replace", "path": "/spec/enableCephTools", "value": true }]'

Reply:
Thx for your comment.
I have updated it with new command.
Pls let me know if it ok.

Reply:
Yeah now the patch command looks good to me now.